AJOAuth2/ViewControllers/LeftViewController.m

```diff
@@ -63,11 +64,11 @@ - (void)viewWillAppear:(BOOL)animated {
         // Email Address
         _emailLabel.text = user.emailAddress;
         
-        // ImageView
-        if (user.avatarImageURLString.length > 0)
-            _avatarImageView.image = [UIImage imageWithData:[Helper avatarImageUrl:user.avatarImageURLString]];
-        else
-            _avatarImageView.image = [UIImage imageNamed:@"circle-user"];
+        // Avatar ImageView        
+        [_avatarImageView setShowActivityIndicatorView:YES];
+        [_avatarImageView setIndicatorStyle:UIActivityIndicatorViewStyleGray];
+        [_avatarImageView sd_setImageWithURL:[NSURL URLWithString:[NSString stringWithFormat:@"%@%@", HOST_URL, user.avatarImageURLString]] placeholderImage:PLACEHOLDER_IMAGE options:SDWebImageRefreshCached];
```

Comment:
[NSURL URLWithString:[NSString stringWithFormat:@"%@%@", HOST_URL, user.avatarImageURLString]]
move to NSURL *url =  to avoid lengthy statement